setup.py

```diff
@@ -1,27 +1,36 @@
 #!/usr/bin/env python
 
-try:
-    from setuptools import setup, find_packages
-except ImportError:
-    from ez_setup import use_setuptools
-    use_setuptools()
-    from setuptools import setup, find_packages
+from __future__ import absolute_import
+from setuptools import setup, find_packages
 
 setup(
     name="Django Autosave",
-    version="0.7.9",
+    version="0.8.0",
     author='Jason Goldstein',
     author_email='jason@betheshoe.com',
     url='https://github.com/theatlantic/django-autosave',
-    packages=['autosave', ],
+    packages=['autosave'],
     description='Generic autosave for the Django Admin.',
     long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
+    install_requires=['Django>=1.11'],
```

Comment:
Since this effectively drops support for Django 1.8, should we do a major version bump and note this in the README?